ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -19,7 +18,11 @@
     <category field="social_science"/>
     <category field="philosophy"/>
     <category field="literature"/>
-    <updated>2016-11-17T16:34:49+00:00</updated>
+    <category field="history"/>
+    <category field="humanities"/>
+    <category field="political_science"/>
+    <category field="sociology"/>    
```

Comment:
@smarchand, this is still rather extreme, though. Could you please trim it back to 2-3 fields max? For comparison, of our 1435 independent CSL styles, just 5 have 4 fields assigned, and only one styles 5 fields. We currently have no styles with more assigned fields than that, while here you go all the way to 7.

In general, if a style can't be assigned to a select few fields, we assign it either no field at all or the "generic-base" field.